### PR TITLE
Cleaning up the code and images for public consumption

### PR DESCRIPTION
**Pull request by Sfshaza**
Code/images for the layout tutorial. These are linked to from the tutorial.

@abarth, could you look through these? Feedback I received from the Flutter Hackathon today is that they want to be able to link to the Dart code and image files. I am linking to the raw code files and image directories on this repo.  thx!

### CODE REVIEW

**Comment by Sfshaza** [comment]
Ugh! I was trying NOT to also update the layout tutorial, but the new code structure broke a link.

File: _includes/_code/column/main.dart [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import 'package:flutter/material.dart';
-import 'package:flutter/rendering.dart' show debugPaintSizeEnabled;
+//import 'package:flutter/rendering.dart' show debugPaintSizeEnabled;
 
 void main() {
-  debugPaintSizeEnabled = true;
+  //debugPaintSizeEnabled = true;
```
Comment (abarth) [inline _includes/_code/column/main.dart]: Are you sure you want this commented out code?

**Comment by abarth** [comment]
LGTM

File: _includes/_code/column/main.dart (hunk 1 above)
Comment (Sfshaza) [inline _includes/_code/column/main.dart]: Well, I kinda did, but I could be swayed.

**Comment by Sfshaza** [comment]
THanks, @abarth, for your late night review!

File: _includes/_code/column/main.dart (hunk 1 above)
Comment (abarth) [inline _includes/_code/column/main.dart]: If you keep it, I'd probably expand it to include a sentence or two explaining what it does and why you might want to try it out.

**Comment by Sfshaza** [comment]
Actually, I removed it from the column example. It will go into the next PR.

thx

**Pull request merged**